jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ClientConnector.java

```diff
@@ -753,4 +764,11 @@ public ChannelWithAddress newChannelWithAddress(ClientConnector clientConnector,
             };
         }
     }
+
+    public interface ClientConnectionListener
```

Comment:
Rename to just `ConnectListener`.

As this is low-level, we do not use the word `Connection`, to avoid confusion with the Jetty class.